ledger/participant-integration-api/src/main/scala/platform/indexer/IndexerStartupMode.scala

```diff
@@ -17,4 +17,6 @@ object IndexerStartupMode {
 
   case object MigrateOnEmptySchemaAndStart extends IndexerStartupMode
 
+  case object EnforceEmptySchemaAndMigrate extends IndexerStartupMode
```

Comment:
Would be nice to add some documentation on the exact difference between these startup modes. I think the names are pretty self-explanatory already but it's difficult to grasp the small difference between `MigrateOnEmptySchemaAndStart` and `EnforceEmptySchemaAndMigrate` without mentally desiccating the names for a while.

Reply:
Do we still use all of these startup modes?